app/helpers/sessions_helper.rb

```diff
@@ -0,0 +1,13 @@
+module SessionsHelper
+  def login(user)
+    session[:user_id] = user.id
+  end
+
+  def current_user
+    @current_user ||= User.find_by(id: session[:user_id])
+  end
+
+  def logged_in?
+    !@current_user.nil?
```

Comment:
i'd be careful not to use nil. another ruby idiom you can use here is: `!!@current_user`
it will be false if @current_user is nil OR FALSE (which aren't the same), and true otherwise.

Reply:
Thank you for the suggestion I will make the change